aiobotocore/awsrequest.py

```diff
@@ -9,8 +14,12 @@ async def _content_prop(self):
         """Content of the response as bytes."""
 
         if self._content is None:
+            if httpx is not None and isinstance(self.raw, httpx.Response):
```

Comment:
This adds runtime overhead (`isinstance` is fairly expensive, especially for older Python versions) when httpx is installed, but not used. Can't we check i.e. a config flag or something? We could then get rid of the import workaround, too.

Reply:
Sure, that would not be an improvement for sure. I was thinking we might get around to dynamically checking types, e.g. if we had a smarter type hierarchy:
As I understand, you are proposing `HttpxSession` as part of this PR. That is where relevant instances of `AioAWSResponse` would be created. Why don't we simply subclass `AioAWSResponse` and override `._content_prop`?

Reply:
oh yeah that works